pipelines/azure-pipelines-nuget.yml

```diff
@@ -0,0 +1,388 @@
+                },
+                {
+                  "parameterName": "OpusInfo",
+                  "parameterValue": "http://www.microsoft.com"
+                },
+                {
+                  "parameterName": "FileDigest",
+                  "parameterValue": "/fd \"SHA256\""
+                },
+                {
+                  "parameterName": "PageHash",
+                  "parameterValue": "/NPH"
+                },
+                {
+                  "parameterName": "TimeStamp",
+                  "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                }
+              ],
+              "toolName": "sign",
+              "toolVersion": "1.0"
+            },
+            {
+              "keyCode": "CP-230012",
+              "operationSetCode": "SigntoolVerify",
+              "parameters": [ ],
+              "toolName": "sign",
+              "toolVersion": "1.0"
+            }
+          ]
+
+    - task: DownloadPipelineArtifact@2
+      displayName: 'Download Windows nuspec and license'
+      inputs:
+        artifactName: WIN32
+        itemPattern: |
+          Package.nuspec
+          build/**
+        targetPath: 'package_win/'
+
+    - script: |
+        xcopy /E $(ArtifactsFolder)\WIN32                     package_win\build\native\WIN32\
+        xcopy /E $(ArtifactsFolder)\WIN32-nobundle            package_win\build\native\WIN32-nobundle\
+        xcopy /E $(ArtifactsFolder)\WIN32-pack                package_win\build\native\WIN32-pack\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64                 package_win\build\native\WIN32-x64\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64-pack            package_win\build\native\WIN32-x64-pack\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64-nobundle        package_win\build\native\WIN32-x64-nobundle\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64chk              package_win\build\native\WIN32-x64chk\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64chk-pack         package_win\build\native\WIN32-x64chk-pack\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64ValidationParser package_win\build\native\WIN32-x64ValidationParser\
+        xcopy /E $(ArtifactsFolder)\WIN32-x64Xerces           package_win\build\native\WIN32-x64Xerces\
+        xcopy /E $(ArtifactsFolder)\WIN32chk                  package_win\build\native\WIN32chk\
+        xcopy /E $(ArtifactsFolder)\WIN32chk-pack             package_win\build\native\WIN32chk-pack\
+        xcopy /E $(ArtifactsFolder)\WIN32ValidationParser     package_win\build\native\WIN32ValidationParser\
+        xcopy /E $(ArtifactsFolder)\WIN32Xerces               package_win\build\native\WIN32Xerces\
+      displayName: 'Prepare Windows'
+
+    - task: NuGetCommand@2
+      displayName: 'NuGet pack Windows'
+      inputs:
+        command: pack
+        packagesToPack: 'package_win\Package.nuspec'
+        packDestination: '$(NuGetsFolder)\windows'
+
+    # Global tasks

```

Comment:
># Global tasks [](http://example.com/codeflow?start=4&length=14)

Can you add a comment that this is just a fail safe to make sure the artifacts come from the same build